scripts/generate_specs.sh

```diff
@@ -30,7 +30,7 @@ case $build_step in
     ;;
   build-testnet)
     echo "Building Spec for frequency as rococo testnet"
-    $PWD/target/release/frequency build-spec --chain=frequency_rococo --disable-default-bootnode > ./res/genesis/testnet/frequency-spec-rococo-testnet.json
+    $PWD/target/release/frequency build-spec --chain=frequency-rococo --disable-default-bootnode > ./res/genesis/testnet/frequency-spec-rococo-testnet.json
```

Comment:
I still cannot track when to use `-` and when `_`. So a ping to double check this line.

Reply:
--chain "{spec-name}" triggers this part of cli, so i am going to change to following

1. "dev" | "rococo-2000" |"" i.e. either of these tags will point to local specs
2. "rococo-4044" | "testnet" will point to testnet specs, and 
3 "frequency" is mainnet 

https://github.com/LibertyDSNP/frequency/blob/1d907b077151c0649f3f6ae7ba56e2b290de09f0/node/cli/src/command.rs#L51